config.test.yaml

```diff
@@ -265,7 +265,7 @@ test:
   content_types:
     html: text/html;profile="mediawiki.org/specs/html/1.1.0";charset=utf-8
     data-parsoid: application/json;profile="mediawiki.org/specs/data-parsoid/0.0.1"
-    wikitext: text/plain;profile="mediawiki.org/specs/wikitext/1.0.0";charset=utf-8
+    wikitext: text/plain; charset=utf-8; profile="mediawiki.org/specs/wikitext/1.0.0"
```

Comment:
No, it's not me reordering it, it's express.  It would require an upstream patch to express.